### ISSUE TITLE
Buttons not clickable emptyState

### ISSUE THREAD

**Issue by pfo-omicsstudio** [icflorescu/mantine-datatable#663]
**Describe the bug**
In the DataTable component, when using the emptyState prop to display a custom empty state containing any interactive button, the button becomes unresponsive. However, if the same component is rendered directly instead of within emptyState, the button works as expected. This appears to be an issue with DataTable’s handling of the emptyState prop.

**To Reproduce**
1. Use the DataTable component with emptyState to render an interactive component containing a button (e.g., Button or ActionIcon).
2. Attempt to click the button in the rendered empty state.
3. The button does not respond to clicks within emptyState.
4. Move the same component outside of DataTable’s emptyState and observe that the button becomes responsive.

Sample code of what I'm referring to:
```tsx

function ExampleTable({ data, openModal }) {
  // Direct rendering example for comparison (works as expected)
  if (data.length === 0) {
    return (
      <Group justify="center" gap="lg">
        <ActionIcon size={24} onClick={openModal}>
          <IconPlus />
        </ActionIcon>
        <Stack align="left" gap={0}>
          <Text c="dimmed" size="sm">
            No records found.
          </Text>
          <Text c="dimmed" size="sm">
            Click the + button to add one
          </Text>
        </Stack>
      </Group>
    );
  }

  // DataTable rendering using emptyState (button does not work)
  return (
    <DataTable
      records={data}
      columns={[
        { accessor: "name", title: "Name" },
        { accessor: "age", title: "Age" }
      ]}
      emptyState={
        <Group gap="lg">
          <ActionIcon size={24} onClick={openModal}>
            <IconPlus />
          </ActionIcon>
          <Stack align="left" gap={0}>
            <Text c="dimmed" size="sm">
              No records found.
            </Text>
            <Text c="dimmed" size="sm">
              Click the + button to add one
            </Text>
          </Stack>
        </Group>
      }
    />
  );
}
```

**Expected behavior**
Buttons within the emptyState prop of DataTable should respond to clicks.

**Screenshots**
Using emptyState prop:

https://github.com/user-attachments/assets/23016287-f5c3-4e6c-855f-c5e8d5c4920f

Rendering directly with early return:

https://github.com/user-attachments/assets/f642c3ed-c0fb-4e7a-8a38-2743b1267b0d

Gyazo links in case embeds stop working:
https://gyazo.com/3c8cd0879ccdba8665433d2d40124384
https://gyazo.com/a7d9577f0c29bbb51be265280ba03622

**Desktop (please complete the following information):**
- OS: MacOS Sonoma 14.6.1
- Google Chrome
- Version 130.0.6723.69 (Official Build) (arm64)

**Comment by icflorescu** [comment]
Thanks for reporting!
This was addressed in 7.14.3, which should hit the npm registry in a few minutes.

**Comment by icflorescu** [comment]
@mariansimecek
Oops, you're right. Will revert in a minute.

**Comment by icflorescu** [comment]
@mariansimecek Fixed in `7.14.4`. Apologies for the inconvenience.
@pfo-omicsstudio To add interactive empty state content, make adjust the styling as explained here: https://icflorescu.github.io/mantine-datatable/examples/empty-state/#interactive-custom-empty-state-content
